lib/emque/producing/message/message.rb

```diff
@@ -64,6 +76,10 @@ def message_type
         self.class.read_message_type
       end
 
+      def raise_message_failure
+        self.class.read_raise_message_failure
```

Comment:
It does default to true in the class method: https://github.com/teamsnap/emque-producing/pull/26/files#diff-eaa13cbc2d2da6e4e638a20047ce0b4dR31